### PR TITLE
Complete functionality for take_turns

### PR DESCRIPTION
**Pull request by suzkiee**
## What does this change?

take_turns method

## What does this fix?

Allows game to loop computer turns and user turns until there is a winner. 

## Is this a bug or a feature? 

feature 

## How was this tested?

We used a test runner file and ran the game in the terminal

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -2,8 +2,6 @@ class Game
 
   # take_turns method
```
Comment (mkrumholz) [inline lib/game.rb:3]: yay this is done!

File: lib/game.rb [hunk 2]
```diff
@@ -2,8 +2,6 @@ class Game
...
 
-  #check_for_winner method
-
   # celebrate_winner
```
Comment (mkrumholz) [inline lib/game.rb:5]: still need to create this one in the next work session, as well as game.play

File: lib/game.rb [hunk 3]
```diff
@@ -39,31 +45,48 @@ def main_menu
...
+    end
+  end
+
   def complete_computer_setup
     submarine = Ship.new("Submarine", 2)
     cruiser = Ship.new("Cruiser", 3)
     ships = [submarine, cruiser]
     computer_setup = Setup.new(@computer_board, ships, :computer)
-    computer = computer_setup.run_setup
+    # computer = computer_setup.run_setup
```
Comment (mkrumholz) [inline lib/game.rb:66]: We are going to need to uncomment these

File: lib/game.rb [hunk 4]
```diff
@@ -39,31 +45,48 @@ def main_menu
...
   end
 
   def complete_user_setup
     submarine = Ship.new("Submarine", 2)
     cruiser = Ship.new("Cruiser", 3)
     ships = [submarine, cruiser]
     user_setup = Setup.new(@user_board, ships, :human)
-    user = user_setup.run_setup
+    # user = user_setup.run_setup
```
Comment (mkrumholz) [inline lib/game.rb:74]: this one too

File: lib/turn.rb [hunk 5]
```diff
@@ -1,16 +1,28 @@
...
+    @user        = user
+    @player_type = player_type
   end
 
-  def display_boards
-    puts "=============COMPUTER BOARD============="
-    puts computer.render
-    puts "==============PLAYER BOARD=============="
-    puts user.render(true)
+  # def display_boards
```
Comment (mkrumholz) [inline lib/turn.rb:13]: I think we can delete these comments

File: lib/game.rb (hunk 1 above)
Comment (suzkiee) [reply lib/game.rb:3]: 🎉 Snaps to your big brain moments on this!! 🎉

File: spec/turn_spec.rb [hunk 6]
```diff
@@ -15,10 +16,11 @@
       computer_setup = Setup.new(board, ships, :computer)
       computer = computer_setup.run_setup
 
-      user_setup = Setup.new(board, ships, :human)
-      user = user_setup.run_setup
+      # user_setup = Setup.new(board, ships, :human)
```
Comment (mkrumholz) [inline spec/turn_spec.rb:19]: Let's come back to this after class

File: lib/game.rb (hunk 2 above)
Comment (suzkiee) [reply lib/game.rb:5]: Yes! And we need to wrap all our helper methods up in the play method

**Pull request merged**